### PR TITLE
[httpjson] Add support for `chain` property

### LINKED ISSUE #4845: [httpjson/Custom API] Support `chain` configuration
## Context

The httpjson/Custom API integration supports `[](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-httpjson.html#chain)` in Filebeat, but it doesn't support it via Fleet management, as the Integration here doesn't seem to have it defined.

It would be nice to be able to use this feature for Fleet managed custom API integrations.

## Use-case

See: https://github.com/elastic/beats/issues/31117

### PR DESCRIPTION
**Pull request by indygriffiths**
## What does this PR do?

Updates `httpjson` with support for setting the [`chain`](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-httpjson.html#chain) property within Kibana.

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues
- Closes #4845

### CODE REVIEW

File: packages/httpjson/changelog.yml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+- version: "1.7.1"
```
Comment (efd6) [inline packages/httpjson/changelog.yml:1]: Please bump this to 1.8.0 as the change is an enhancement.

**Comment by rsdrakh** [comment]
@efd6 Yeah, I really support that thing @philippkahr said :-)

**Review by efd6 — APPROVED** [review]
Apologies for the delay.

**Pull request merged**